src/generator/operations.ts

```diff
@@ -438,6 +447,14 @@ function createProtocolRequest(codeModel: CodeModel, client: string, op: Operati
         text += '\tunencodedParams := []string{}\n';
       }
       for (const qp of values(unencodedParams)) {
+        let setter: string;
+        if (qp.protocol.http?.explode === true) {
+          setter = `\tfor _, qv := range ${getParamName(qp)} {\n`;
+          setter += `\t\tunencodedParams = append(unencodedParams, "${qp.language.go!.serializedName}="+qv)\n`;
```

Comment:
Just want to know what the = for after `${qp.language.go!.serializedName}`? I dont think I saw it in any of the testserver examples.

Reply:
When exploding a query param `queryParam` with the following values `foo,bar,baz` we want to generate the following sequence.

`queryParam=foo&queryParam=bar&queryParam=baz`

You are right there's no codegen with this as we don't have a test server example that uses exploded, unencoded query params.

Reply:
Ok got it 👍 yeah we should have something that tests this in particular